Exercises/exercise_3_join_validator.md

```diff
@@ -1,27 +1,108 @@
 # Exercise 3
 Join and leave the Axelar Network as a validator node.
 
-Convert an existing Axelar Network node into a validator by staking tokens. A validator participates in block creation, transaction signing, and voting.
+Convert an existing Axelar Network node into a validator by attaching Bitcoin and Ethereum nodes and staking tokens. A validator participates in block creation, transaction signing, and voting.
 
 ## Level
-Intermediate
+Advanced
 
 ## Disclaimer
 Axelar Network is a work in progress. At no point in time should you transfer any real assets using Axelar. Only use testnet tokens that you're not afraid to lose. Axelar is not responsible for any assets lost, frozen, or unrecoverable in any state or condition. If you find a problem, please submit an issue to this repository following the template.
 
 ## Prerequisites
 - Complete all steps from `README.md`
-- RPC endpoints for your Bitcoin and Ethereum nodes
 
 ## Useful links
 - Extra commands to query Axelar Network state: https://github.com/axelarnetwork/axelarate-community/blob/main/EXTRA%20COMMANDS.md
 
 ## Joining the Axelar testnet
 Follow the instructions in `README.md` to make sure your node is up to date and you received some test coins to your validator account.
 
-## Set up Bitcoin and Ethereum RPC nodes
-As an Axelar Network validator, your Axelar node will vote on the status of Bitcoin and Ethereum transactions. To do so, it needs to be configured with your Bitcoin and Ethereum nodes using the RPC endpoint.
+## Set up Bitcoin and Ethereum nodes
+As an Axelar Network validator, your Axelar node will vote on the status of Bitcoin and Ethereum transactions. To do so, you must first set up and configure your Bitcoin and Ethereum testnet nodes, and then provide the RPC endpoints to Axelar.
 
+If you do not already have a Bitcoin testnet node and Ethereum Ropsten testnet node running, follow the instructions in `BTC ETH NODE SETUP.md` to set up and configure them, then skip to the next section `Connect Bitcoin and Ethereum nodes to Axelar`.
```

Comment:
"you can follow instructions in BTC ETH ... , or follow an alternative configuration of your choice"